src/main/java/se/sundsvall/digitalmail/api/DigitalMailResource.java

```diff
@@ -64,14 +65,16 @@ class DigitalMailResource {
         produces = { APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE }
     )
     ResponseEntity<DigitalMailResponse> sendDigitalMail(@Valid @RequestBody final DigitalMailRequest request) {
-        // Validate body as HTML
-        if (TEXT_HTML_VALUE.equals(request.getBodyInformation().getContentType()) && !htmlValidator.validate(request.getBodyInformation().getBody())) {
-            throw Problem.builder()
-                .withTitle("Body HTML is invalid")
-                .withStatus(BAD_REQUEST)
-                .withDetail("Use https://validator.w3.org/ to make sure your HTML validates")
-                .build();
-        }
+
+        Optional.ofNullable(request.getBodyInformation())
+                .filter(bodyInfo -> TEXT_HTML_VALUE.equals(request.getBodyInformation().getContentType()) && !htmlValidator.validate(request.getBodyInformation().getBody()))
```

Comment:
Should be `...equals(bodyInfo.getContentType())` and so on, right? To avoid potential NPE:s